ldes-server-infra-mongo/mongock-changeset-1/pom.xml

```diff
@@ -34,13 +34,24 @@
         </dependency>
     </dependencies>
 
+
     <build>
         <finalName>${project.artifactId}</finalName>
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-assembly-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <exclude>be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/mongock/changeset1/*</exclude>
```

Comment:
would it be possible to move the wildcard one level higher? Then all future changesets are excluded as well